Doc/source/merge.rst

```diff
@@ -1,8 +1,10 @@
-#####
-merge
-#####
+####################################
+merge: Merge multiple fonts into one
+####################################
 
-.. automodule:: fontTools.merge
+``fontTools.merge`` provides both a library and a command line interface
+(``pyftmerge``) for merging multiple fonts together.
```

Comment:
I would like eventually to get rid of the `pyft*` scripts and encourage people to use the `fonttools {subset,merge,...}` entry point. Maybe just mention `fonttools merge` here. We can decide to deprecate them for good